docs/nodes/nodes-library/trigger-nodes/TypeformTrigger/README.md

```diff
@@ -24,9 +27,77 @@ The final workflow should look like the following image.
 
 ### 1. Typeform Trigger node
 
+This node will trigger the workflow when a form response is submitted.
+
 1. First of all, you'll have to enter credentials for the Typeform Trigger node. You can find out how to do that [here](../../../credentials/Typeform/README.md).
-2. Select the form you want to receive the answers from in the *Form* field.
-3. Click on *Execute Node* to run the workflow.
+2. Select your form from the ***Form*** dropdown list.
+3. Click on ***Execute Node*** to run the workflow.
+
+In the screenshot below, you will notice that the Typeform Trigger node triggers the workflow and returns the response submitted by a user.
+
+![Using the Typeform Trigger node to trigger the workflow](./TypeformTrigger_node.png)
+
+### 2. Set node
+
+We will use the Set node to ensure that only the data that we set in this node gets passed on to the next nodes in the workflow. We will set the value of `Name`, `Email`, `Interest`, and `Country` in this node.
+::: v-pre
+1. Click on ***Add Value*** and select 'String' from the dropdown list.
+2. Enter `Name` in the ***Name*** field.
+3. Click on the gears icon next to the ***Value*** field and click on ***Add Expression***.
+4. Select the following in the ***Variable Selector*** section: Current Node > Input Data > JSON > Let's start with your *name.*. You can also add the following expression: `{{$json["Let's start with your *name.*"]}}`.
+5. Click on ***Add Value*** and select 'String' from the dropdown list.
+6. Enter `Email` in the ***Name*** field.
+7. Click on the gears icon next to the ***Value*** field and click on ***Add Expression***.
+8. Select the following in the ***Variable Selector*** section: Current Node > Input Data > JSON > [field:d566770d2197a78b] what's your email address?. You can also add the following expression: `{{$json["[field:d566770d2197a78b] what's your email address?"]}}`.
```

Comment:
We should avoid these in examples as this expression might not work for other users.